ethereum/eth/src/test/java/tech/pegasys/pantheon/ethereum/eth/sync/tasks/GetHeadersFromPeerByNumberTaskTest.java

```diff
@@ -97,7 +101,13 @@ private void getHeadersFromHash(final int skip, final boolean reverse) {
     // Execute task and wait for response
     final AbstractGetHeadersFromPeerTask task =
         new GetHeadersFromPeerByNumberTask(
-            protocolSchedule, ethContext, startNumber, count, skip, reverse);
+            protocolSchedule,
+            ethContext,
+            startNumber,
+            count,
+            skip,
+            reverse,
+            labels -> () -> () -> 0.0D);
```

Comment:
Should we make this a constant somewhere? It's a very cryptic line to have scattered around the code base so much.